wolfssl/wolfcrypt/asn_public.h

```diff
@@ -244,7 +244,11 @@ typedef struct WOLFSSL_ASN1_TIME {
 
 enum {
     IV_SZ   = 32,                   /* max iv sz */
+#ifdef OPENSSL_ALL
```

Comment:
Sorry, can you add an inline code comment for why this is being increased? It looks like for the maximum password length. I am curious what specifically is driving this.

Reply:
Yeah will do. But basically : feature request list, item 29

Reply:
Comment added :)